events/events.go

```diff
@@ -127,9 +127,12 @@ func reconcileResponderEvent(tx *gorm.DB, event api.Event) error {
 	var externalID string
 	if responder.Properties["responderType"] == "Jira" {
 		externalID, err = responderPkg.NotifyJiraResponder(responder)
-		if err != nil {
-			return err
-		}
+	} else if responder.Properties["responderType"] == "MSPlanner" {
```

Comment:
Can we make these `MSPlanner` and JIRA as constants and probably create a type Responders which is a string and restrict the types from there?

Reply:
👍🏽